docs/EVG.md

```diff
@@ -1320,27 +1324,27 @@ A CA may rely on a previously verified certificate request to issue a replacemen
 
 ### 11.14.3.  Age of Validated Data
 
-(1) Except for reissuance of an EV Certificate under Section 11.14.2 and except when permitted otherwise in Section 11.14.1, the age of all data used to support issuance of an EV Certificate (before revalidation is required) SHALL NOT exceed the following limits:
+(1) In addition to the requirements on the reuse of data set forth within the Baseline Requirements, the age of all data used to support the issuance of an EV Certificate MUST NOT exceed the following limits, except in the re-issuance of an EV Certificate under Section 11.14.2:
```

Comment:
@pzb I’m not sure I follow why you suggest that all of 11.14.3 could be ignored. As originally written here, it clearly only applies to 11.14.3 (1), which is still the property maintained.

You are correct to understand that part of the goal is to ensure that this does not supersede or replace the requirements set forth in the Baseline Requirements.

Could you help me understand which clause of (1) would prevent rekey, where it should be permitted now, and without conflicting with the BRs?